opennms-base-assembly/src/main/filtered/etc/events/opennms.internal.events.xml

```diff
@@ -230,4 +230,19 @@
         </logmsg>
       <severity>Normal</severity>
    </event>
+   <event>
+      <uei>uei.opennms.org/internal/telemetry/clockSkewDetected</uei>
+      <event-label>Clock Skew detected</event-label>
+      <descr>Clock skew detected for flow exporter</descr>
+      <logmsg dest="logndisplay">Clock skew for exporter with interface '%interface%' in location '%parm[monitoringSystemLocation]%' detected by '%parm[monitoringSystemId]%'.</logmsg>
+      <severity>Warning</severity>
+   </event>
+   <event>
+      <uei>uei.opennms.org/translator/telemetry/clockSkewDetected</uei>
+      <event-label>Clock Skew detected</event-label>
+      <descr>Clock skew detected for flow exporter</descr>
```

Comment:
Can we include both the delta and the configured limits in the description?

Reply:
done